packages/components/form-control/src/form-control.tsx

```diff
@@ -425,8 +427,10 @@ export const RequiredIndicator = forwardRef<RequiredIndicatorProps, "span">(
         __css={css}
         {...rest}
       >
-        *
+        {children ? children : <>*</>}
```

Comment:
Change `{children ? children : <>*</>}` to `{children ?? <>*</>}`

Reply:
I fixed!